src/placeos-frontend-loader/api/remote/githubremote.cr

```diff
@@ -9,15 +9,12 @@ module PlaceOS::FrontendLoader
 
     private alias Remote = PlaceOS::FrontendLoader::Remote
 
-    TAR_NAME = "temp.tar.gz"
-
     @github_client = Octokit.client(GIT_USER, GIT_PASS)
 
     # Returns the branches for a given repo
     def branches(repo : String) : Hash(String, String)
-      get_branches = @github_client.branches(repo)
       branches = Hash(String, String).new
-      get_branches.fetch_all.each do |branch|
+      @github_client.branches(repo).fetch_all.map do |branch|
         next if branch =~ /HEAD/
         branches[branch.name] = branch.commit.sha
       end
```

Comment:
The regex match appears to be on the branch object, should it be on the name? or the reference?

```suggestion
      @github_client.branches(repo).fetch_all.each_with_object({} of String => String) do |branch, branches|
        next if branch =~ /HEAD/
        branches[branch.name] = branch.commit.sha
      end
```